### PR TITLE
GA tracker to export and reset

### PR DESCRIPTION
**Pull request by Georgehe4**
@lukechurch #617 
Added analytics to pressing the reset/export and affirming them.

### CODE REVIEW

File: lib/embed/embed.dart [hunk 1]
```diff
@@ -219,14 +219,20 @@ class PlaygroundMobile {
   void registerExportButton() {
     if ($('[icon="launch"]') != null) {
       _exportButton = new PaperIconButton.from($('[icon="launch"]'));
-      _exportButton.clickAction(_exportDialog.toggle);
+      _exportButton.clickAction(() {
+        _exportDialog.toggle;
```
Comment (lukechurch) [inline lib/embed/embed.dart]: I think these are methods that need to be called() ?

**Comment by lukechurch** [comment]
lgtm

**Comment by Georgehe4** [comment]
@lukechurch 
Interestingly enough, as long as a button has an onclick call, Polymer seems to deal with most of the toggling interactions for closing the dialog.

**Comment by Georgehe4** [comment]
Merging. May revisit if polymer elements change behavior.

**Pull request merged**